packages/flutter_markdown/lib/src/_functions_web.dart

```diff
@@ -50,20 +51,15 @@ final MarkdownStyleSheet Function(BuildContext, MarkdownStyleSheetBaseTheme?)
   BuildContext context,
   MarkdownStyleSheetBaseTheme? baseTheme,
 ) {
-  MarkdownStyleSheet result;
-  switch (baseTheme) {
-    case MarkdownStyleSheetBaseTheme.platform:
-      final String userAgent = window.navigator.userAgent;
-      result = userAgent.contains('Mac OS X')
-          ? MarkdownStyleSheet.fromCupertinoTheme(CupertinoTheme.of(context))
-          : MarkdownStyleSheet.fromTheme(Theme.of(context));
-    case MarkdownStyleSheetBaseTheme.cupertino:
-      result =
-          MarkdownStyleSheet.fromCupertinoTheme(CupertinoTheme.of(context));
-    case MarkdownStyleSheetBaseTheme.material:
-    default: // ignore: no_default_cases
-      result = MarkdownStyleSheet.fromTheme(Theme.of(context));
-  }
+  final MarkdownStyleSheet result = switch (baseTheme) {
+    MarkdownStyleSheetBaseTheme.platform when _userAgent.contains('Mac OS X') =>
+      MarkdownStyleSheet.fromCupertinoTheme(CupertinoTheme.of(context)),
+    MarkdownStyleSheetBaseTheme.cupertino =>
+      MarkdownStyleSheet.fromCupertinoTheme(CupertinoTheme.of(context)),
+    MarkdownStyleSheetBaseTheme.material ||
```

Comment:
nit: unnecessary, right?

Reply:
yeah, trying to keep the existing...whatever...